libraries/apollo-websocket-network-transport-incubating/src/jvmMain/kotlin/com/apollographql/apollo3/network/ws/incubating/DefaultOkHttpClientBuilder.kt

```diff
@@ -0,0 +1,8 @@
+package com.apollographql.apollo3.network.ws.incubating
+
+
+import okhttp3.OkHttpClient
+
+internal val defaultOkHttpClientBuilder: OkHttpClient.Builder by lazy {
```

Comment:
Why not just calling `OkHttpClient.Builder()` on demand?

Reply:
I want to share the OkHttp threadpool between WebSockets and HTTP. So this is technically not used right now but I'll enable it when swapping the websocket network transport is not incubating anymore.